app/src/main/java/de/westnordost/streetcomplete/data/tiles/DownloadedTilesDao.java

```diff
@@ -70,6 +70,12 @@ public int remove(Point tile)
 				DownloadedTilesTable.Columns.Y + " = ?", whereArgs);
 	}
 
+	public void removeAll()
+	{
+		SQLiteDatabase db = dbHelper.getReadableDatabase();
```

Comment:
get**Writable**Database

Reply:
But you are doing the same here:
https://github.com/westnordost/StreetComplete/blob/fb3f6ccddf64e35ecb1576a7cb13471474d775f5/app/src/main/java/de/westnordost/streetcomplete/data/visiblequests/VisibleQuestTypeDao.java#L99